src/acquisition/covidcast/data_dir_readme.md

```diff
@@ -0,0 +1,35 @@
+# covidcast data directory
+
+NOTE: Edit this file on GitHub!
```

Comment:
Is this a lingering TODO or some weird thing where editing locally is ill-advised?

Reply:
this file gets deployed to `/common/covidcast/README.md`. the note is there to prevent someone from editing that file directly on the server, only to have their changes clobbered the next time this repo gets deployed.

Reply:
to clarify: editing locally is no problem so long as the change is committed to the repo